hw/ip/i2c/data/i2c.hjson

```diff
@@ -61,6 +76,13 @@
                 Enable Host I2C functionality
                 '''
         }
+        { bits: "1"
+          resval: "0"
+          name: "ENABLETARGET"
```

Comment:
Typically, excluding the main control bit from writes is a good idea - it helps to avoid adding exclusions in several other places.

Reply:
Do you mean apply "CsrExclWrite" for these main control bits?

Reply:
Yeah, the "excl:CsrNonInitTests:CsrExclWrite" exclusion. Do this only if it helps remove exclusions in other places. Otherwise nothing to do here.